client/extensions/iptc/src/extension.ts

```diff
@@ -36,16 +38,38 @@ const extension: IExtension = {
                                     translations: subj.translations,
                                 });
                             });
-                    }
+                    });
 
                     Object.assign(item, {
+                        language: 'nl',
```

Comment:
Don't mutate the item that's being passed as an argument. Instead, you can create another item and mutate that if you need to.

```
const itemNext = {
    ...item,
    language: 'nl',
    // ...
};
```